docs/circuit_cutting/explanation/index.rst

```diff
@@ -1,5 +1,43 @@
-###########################
-Circuit Cutting Explanation
-###########################
+###################################################
```

Comment:
@garrison I think the QPD explanatory can/should go in here, rather than having an extra explanatory. We can just have a QPD section after the overview I think